udata_front/theme/gouvfr/assets/js/locales/fr.json

```
@@ -331,62 +333,60 @@
   "You can manage your datasets on the current admin.": "Vous pouvez gérer vos jeux de données sur l'administration actuelle.",
   "You can edit your profil on the current admin.": "Vous pouvez modifier votre profil sur l'administration actuelle.",
   "You can manage your reuses on the current admin.": "Vous pouvez gérer vos réutilisations sur l'administration actuelle.",
-  "Members": "",
-  "Accept request": "",
-  "Refuse": "",
-  "{n} requests": "",
-  "{n} members": "",
-  "Status": "",
-  "Action": "",
-  "Edit": "",
-  "Edit member": "",
-  "Role of the member": "",
-  "Validate": "",
-  "Remove member from the organization": "",
-  "Be careful, this action can't be reverse.": "",
-  "Remove member": "",
-  "An error occurred while update member role.": "",
-  "Add an image": "",
-  "Paste a link of your image to add it to your description. It is recommended to also provide a textual alternative for an informative image and a title.": "",
-  "Image link": "",
-  "Paste your link...": "",
-  "Customize accessibility options": "",
-  "Image alternative text": "",
-  "Image title": "",
-  "Add image": "",
-  "Image": "",
-  "Undo": "",
-  "Redo": "",
-  "Bold": "",
-  "Italic": "",
-  "Subtitle": "",
-  "Table": "",
-  "Link": "",
-  "List unordered": "",
-  "List ordered": "",
-  "Code block": "",
-  "Quote": "",
-  "Paste link...": "",
-  "Confirm ⏎": "",
-  "Edit link": "",
-  "Remove link": "",
-  "Add row before": "",
-  "Add column before": "",
-  "Delete selected cells": "",
-  "Add row after": "",
-  "Add column after": "",
-  "Align left": "",
-  "Align center": "",
-  "Align right": "",
-  "Membership Request": "",
-  "{fullName} asks to join the organization.": "",
-  "Refuse membership request": "",
-  "You can provide the refusal reason:": "",
-  "Cancel": "",
-  "Refuse request": "",
-  "An error occurred while removing this member.": "",
-  "Pagination": "",
-  "Page {nb}": "",
-  "An error occurred while accepting this membership.": "",
-  "An error occurred while refusing this membership.": ""
-}
\ No newline at end of file
+  "Members": "Membres",
+  "Accept request": "Accepter la demande",
+  "Refuse": "Refuser",
+  "{n} requests": "{n} demandes | {n} demande | {n} demandes",
+  "{n} members": "{n} membres | {n} membre | {n} membres",
+  "Status": "Statut",
+  "Action": "Action",
+  "Edit": "Modifier",
+  "Edit member": "Modifier le membre",
+  "Role of the member": "Rôle du membre",
+  "Validate": "Valider",
+  "Remove member from the organization": "Retirer le membre de l'organisation",
+  "Be careful, this action can't be reverse.": "Attention, cette action ne peut pas être annuler.",
+  "Remove member": "Retirer le membre",
+  "An error occurred while update member role.": "Une erreur s'est produite lors de la mise à jour du rôle du membre.",
```

Je pense qu'on pourrait améliorer l'anglais, mais ça demande de modifier et rééextraire :scream: 
```suggestion
  "An error occurred while updating the member role.": "Une erreur s'est produite lors de la mise à jour du rôle du membre.",
```

---

On peut mettre à jour la traduction en anglais seulement pour cette fois et faire la mise à jour de la clef en meme temps que https://github.com/etalab/data.gouv.fr/issues/1336